fuzzing/fuzzer.go

```diff
@@ -664,15 +711,15 @@ func (f *Fuzzer) printMetricsLoop() {
 		secondsSinceLastUpdate := time.Since(lastPrintedTime).Seconds()
 
 		// Print a metrics update
-		fmt.Printf(
-			"fuzz: elapsed: %s, call: %d (%d/sec), seq/s: %d, resets/s: %d, cov: %d\n",
+		f.logger.Info(fmt.Sprintf(
+			"fuzz: elapsed: %s s, calls: %d (%d/sec), seq/s: %d, resets/s: %d, coverage: %d",
```

Comment:
Do we want this log object to become more colorful? We can make throw it all into a `StructuredLogInfo` as well and that will add some coloring by defualt.

Reply:
I think it'd be nice :)

Reply:
Resolved in [347e76a](https://github.com/crytic/medusa/pull/171/commits/347e76ac1510af28a0345a3e59d34da9a6fb3197)